addons/web_editor/static/src/js/wysiwyg/widgets/link_tools.js

```diff
@@ -481,12 +512,24 @@ export class LinkTools extends Link {
             this.props.wysiwyg.odooEditor.historyStep();
         }
     }
+    _checkDocumentState() {
+        this.state.isDocument = false;
+        this.state.directDownload = true;
```

Comment:
Nitpicking but it's odd to me that this would be set to true if it's not a document. I realise that it doesn't really matter and it probably makes more sense since the media dialog returns a "&download=1" link

Reply:
Yes, by default the link returned is appended with &download=true.